pystan/stan_fit.hpp

```diff
@@ -1094,14 +1158,25 @@ namespace pystan {
 
           if (args.get_ctrl_sampling_metric() == DENSE_E) {
             if (!args.get_ctrl_sampling_adapt_engaged()) {
+              if (args.get_metric_file_flag()) {
               return_code = stan::services::sample
-                ::hmc_static_dense_e(model, *init_context_ptr,
+                ::hmc_static_dense_e(model, *init_context_ptr, *metric_context_ptr,
```

Comment:
do we ignore metric if adapt == false?

Reply:
Need to checkout what CmdStan does

Reply:
https://github.com/stan-dev/cmdstan/blob/develop/src/cmdstan/command.hpp

We don't ignore metric, even if adapt == False.